src/main/java/uk/gov/hmcts/divorce/document/mapper/CaseDataToDraftPetitionTemplateMapper.java

```diff
@@ -0,0 +1,112 @@
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.APPLICANT_2_MIDDLE_NAME;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.CCD_CASE_REFERENCE;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.CHILDREN_OF_THE_APPLICANT_AND_THE_RESPONDENT;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.CIVIL_PARTNERSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.CONDITIONAL_ORDER_OF_DIVORCE_FROM;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.COSTS_RELATED_TO_ENDING_THE_CIVIL_PARTNERSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.COURT_CASE_DETAILS;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.DISSOLUTION_OF_THE_CIVIL_PARTNERSHIP_WITH;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.DIVORCE_COSTS;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.DIVORCE_OR_DISSOLUTION;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.DIVORCE_OR_DISSOLUTION_COST;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.DIVORCE_OR_END_CIVIL_PARTNERSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.FINANCIAL_ORDER_CHILD;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.FINANCIAL_ORDER_OR_DISSOLUTION;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.FOR_A_DIVORCE;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.HAS_COST_ORDERS;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.HAS_FINANCIAL_ORDERS;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.HAS_FINANCIAL_ORDERS_FOR_CHILD;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.ISSUE_DATE;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.MARRIAGE;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.MARRIAGE_DATE;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.MARRIAGE_OR_CIVIL_PARTNERSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.MARRIAGE_OR_RELATIONSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.OF_THE_DIVORCE;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.RELATIONSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.RESPONDENT_POSTAL_ADDRESS;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.TO_END_A_CIVIL_PARTNERSHIP;
+import static uk.gov.hmcts.divorce.document.mapper.DocmosisTemplateConstants.TO_END_THE_CIVIL_PARTNERSHIP;
+
+@Component
+@Slf4j
+public class CaseDataToDraftPetitionTemplateMapper {
```

Comment:
I would call this DraftPetitionTemplateContent and the method apply as it seems to follow the same pattern as CommonContent